client/src/screens/Main/index.js

```diff
@@ -115,7 +115,7 @@ class Inner extends React.PureComponent {
   render() {
         return (
           <Layout>
-            <NavBar />
+            <NavBar isUserSignedIn={true} />
```

Comment:
Maybe we can use the Context API to handle login state

Reply:
yes definitely, this is just a placeholder